### ISSUE TITLE
V-Select triggers change event when click on the already selected option

### ISSUE THREAD

**Issue by newbie95** [vuetifyjs/vuetify#6052]

**Comment by sh7dm** [comment]
You've submitted an issue that appears to have not been created from our [issue creator](https://issues.vuetifyjs.com).
In order to effectively disposition this, we need additional information. This issue will be tagged as `missing information` for the next 14 days before being closed.

Thank you for your contribution and interest in improving Vuetify.
If you have any additional questions, please reach out to us in our [Discord community](https://community.vuetifyjs.com).

**Comment by newbie95** [comment]
https://jsfiddle.net/RaghulRajesh/eywraw8t/531880/ please refer this click on the already selected option triggers change event

**Comment by newbie95** [comment]
I have a fix for this 
line on 659 VSelect.js
``` 
setValue: function setValue(value) {
            if (this.internalValue !== value) {
                this.internalValue = value;
                this.$emit('change', value);
            }
        }```

**Comment by sh7dm** [comment]
@newbie95 Thanks, I have an another fix that I'll make. but your solution is good

**Comment by sh7dm** [comment]
@newbie95 I've opened the PR for this fix. It will be fixed as fast as possible

**Comment by sh7dm** [comment]
@newbie95 not a problem! ;) It's cool that you like Vuetify! If you like to contribute, see [good first issue](https://github.com/vuetifyjs/vuetify/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22) label! If you need help reach out to us in our [Discord community](https://community.vuetifyjs.com) or ask me for help, I'll help you contribute.

**Comment by newbie95** [comment]
how to get permission to push commits

**Comment by sh7dm** [comment]
@newbie95 [guide](https://help.github.com/articles/fork-a-repo/)

**Comment by sh7dm** [comment]
@newbie95 you need to fork repo and create a **NEW BRANCH** out of `master` for bug fixes or `dev` for new features. **Please don't work on `master` or `dev` branch of your fork**. Read more [here](https://github.com/vuetifyjs/vuetify/blob/master/.github/CONTRIBUTING.md).

**Comment by sh7dm** [comment]
Then open the PR.

**Comment by sh7dm** [comment]
PR stands for pull request

**Comment by newbie95** [comment]
Many thanks for the guidlines ❤️ very useful for me 👏

**Comment by sh7dm** [comment]
@newbie95 no problems ;)

**Comment by johnleider** [comment]
fixed in #6054

**Comment by lucafaggianelli** [comment]
I think this created a side effect, I'm using v-autocomplete in v1.4.3 for picking a value and doing something with the selected item, but the v-model is not updated in the @change handler, it shows the previously clicked item.

See this:
https://jsfiddle.net/m206y9nr/

Or am I using the @change and v-model in a bad manner? At the end I fixed it using @input handler
